src/vm/eetwain.cpp

```diff
@@ -5913,6 +5913,14 @@ ULONG32 EECodeManager::GetStackParameterSize(EECodeInfo * pCodeInfo)
     } CONTRACTL_END;
 
 #if defined(_TARGET_X86_)
+#if defined(WIN64EXCEPTIONS)
+    if (pCodeInfo->IsFunclet())
```

Comment:
@janvorli Yes, it was. While digging ``JIT.Methodical.flowgraph.dev10_bug679008.fgloop``, I found that there is a mismatch between ``Esp`` and ``ResumeEsp`` and their difference was the stack argument size on the function itself.